setup.py

```diff
@@ -27,7 +27,7 @@
         "click>=8.0.3",  # used in scl
         "eclipse-sumo==1.10.0",  # sumo
         "gym==0.19.0",
-        "numpy>=1.19.5",
+        "numpy>=1.20.0",

```

Comment:
Why is this needed?

Reply:
It's the second thing I wrote about that in the PR description:  we've been using `sliding_window_view()` in `genhistories.py` for a while and `numpy` has been in `setup.py` for a while without a minimum version.  But recently, we added a minimum version of `numpy` there which is too low such that, if you reinstall and get `1.19.5` (our stated minimum version) then it won't support `sliding_window_view()`.  I encountered this when trying to test the fix above for Patrick.

Reply:
ok, due to issue #1321, I've reverted the `numpy` version to `1.19.5`.